src/liboslexec/llvm_ops.cpp

```diff
@@ -642,14 +642,14 @@ OSL_SHADEOP void osl_transformn_vmv(void *result, void* M_, void* v_)
 {
    const Vec3 &v = VEC(v_);
    const Matrix44 &M = MAT(M_);
-   M.inverse().transpose().multDirMatrix (v, VEC(result));
+   M.inverse().transposed().multDirMatrix (v, VEC(result));
```

Comment:
Would be even nicer to just have `multDirMatrixT` so you don't need to flip the matrix at all.

But I guess this function is rarely called ...